underpants.go

```diff
@@ -174,10 +197,6 @@ type disp struct {
 
   // The OAuth configuration object needed for authentication.
   oauth *oauth.Config
-
-  // An LRU cache that maps raw cookie strings to full user object. This avoids having
-  // to authenticate and decode user cookies on each request.
-  cache *lilcache.Cache
```

Comment:
Honestly, I don't think it would be a viable remote attack vector (although there have been some experiments with promising results when attacking from the same colo) and I didn't spend any time verifying that one existed. 

From my reading of [`mapaccess1_faststr()`](https://golang.org/src/runtime/hashmap_fast.go#L179) it will take longer to return if there is a string with the same prefix in the bucket. I don't know enough about golang internals to know how many buckets a map with as few elements as this one would have, but it just seemed like another good reason to take the cache out if it wasn't giving any speed benefit.